rengobot.py

```diff
@@ -23,7 +23,7 @@
 # raw_input()
 
 intents = discord.Intents.default()
-# intents.message_content = True
+intents.message_content = True
```

Comment:
I had to do to get it to work locally.  I guess you might need to revert if it causes you problems.